v2/manifests/core-ons/dp-frontend-feedback-controller.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  dp-frontend-feedback-controller:
+    x-repo-url: "https://github.com/ONSdigital/dp-frontend-feedback-controller"
+    build:
+      context: ${DP_REPO_DIR:-../../../..}/dp-frontend-feedback-controller
+      dockerfile: Dockerfile.local
```

Comment:
dp-frontend-feedback-controller doesn't look to have a Dockerfile.local

Reply:
I have raised a PR to add this